documentation/asciidoc/stories/assembly_install.adoc

```diff
@@ -0,0 +1,14 @@
+ifdef::context[:parent-context: {context}]
+[id='install']
+:context: install
+= Installing the {brandname} Helm chart
+
+[role="_abstract"]
+Install the Helm chart to build and deploy {brandname} clusters on {openshift}.
```

Comment:
I'm not sure we should be mentioning Openshift specifically here, as in the CMD line section we detail how to install on Vanilla k8s.